expyfun/stimuli/_tracker.py

```diff
@@ -223,6 +237,10 @@ def respond(self, correct):
                     self._n_reversals += 1
                 if self._direction <= 0:
                     self._direction = 1
+        if self._repeat_limit == 'reversal':
```

Comment:
I think our choices here are to:

1. Stay at the same level. This means it counts as a reversal despite no change in direction.
2. Go away from the limit. This means there is a change in direction, but it is in the opposite direction of what is suggested by the behavioral response.

I think I actually prefer (1). It's clearer, then, that changes in direction are only ever caused by a behavioral response that indicates moving in that direction.

I'm not sure how this border behavior works with e.g. 2-up 1-down, though -- I've only been thinking in the context of 1-up 1-down.